Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,24 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+
+function converttopounds(penceString) {
```

Comment:
Notice that the functions you are reading are all written in a specific conventional way. It's called camelCase https://developer.mozilla.org/en-US/docs/Glossary/Camel_case

Reply:
Thank you for the link @SallyMcGrath !
I learnt about Camel-case from that and amended the name of the function.